src/components/Cards/MyTripCard.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from './MyTripCard.module.scss';
+
+export interface MyTripCardProps {
+  trip: {
+    product_id: number;
+    title: string;
+    start_date: string;
+    end_date: string;
+    price: number;
+  };
+}
+
+const formatDate = (dateStr: string) =>
+  dateStr ? `${dateStr.slice(0, 4)}년 ${dateStr.slice(5, 7)}월 ${dateStr.slice(8, 10)}일` : '-';
+
+const getTripDays = (start: string, end: string) =>
+  isNaN(new Date(start).getTime()) || isNaN(new Date(end).getTime())
+    ? '-'
+    : Math.round((new Date(end).getTime() - new Date(start).getTime()) / (1000 * 60 * 60 * 24)) + 1;
```

Comment:
코드가 길고, 중복된 부분이 있어서 삼항연산자가 가독성일 해치는것같군요~
여기서는 중복을 제거하고 early return 하는게 코드는 길어지겠지만, 훨씬 가독성이 있어보여요 

```suggestion
const getTripDays = (start: string, end: string) => {
  const startTime = new Date(start).getTime()
  const endTime = new Date(end).getTime()

  if (isNaN(startTime) || isNaN(endTime)) {
    return '-';
  }
  
  return Math.round((endTime - endTime) / (1000 * 60 * 60 * 24)) + 1;
}
```